CONTRIBUTING.md

```diff
@@ -32,3 +32,20 @@ If you are adding a new file it should have a header like this:
  */
 ```
 
+## Testing
+
+To test, run `npm test`.
+
+## Publishing
+
+This package can be published by members of the Koala team.
+To do this:
+
+ - Add a title with the new version for changes in the `UNRELEASED` section of `CHANGELOG.md`.
+ - Change the version in `package.json` to reflect the new version.
```

Comment:
You can just run `npm version major|minor|patch` and this would version for you and set the tags for git too so you don't need to then do `git tag`

Reply:
Is this an improvement? Would be nice to keep all the release changes in a single commit
https://github.com/Skyscanner/backpack-node-sass/pull/52/files?short_path=eca12c0#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055